frontend/src/components/transactions/ReceiptsOutcomeList.tsx

```diff
@@ -0,0 +1,38 @@
+import * as T from "../../libraries/explorer-wamp/transactions";
+import ReceiptOutcomeRow from "./ReceiptOutcomeRow";
+export interface Props {
+  receipts: T.ReceiptsOutcomeList;
+}
+
+export default ({ receipts }: Props) => {
```

Comment:
please follow new pattern to export component not getting next js warning again

Reply:
@icerove thanks, I fixed it